src/proto/FunctionRpc.proto

```diff
@@ -315,9 +327,18 @@ message TypedData {
     CollectionString collection_string = 9;
     CollectionDouble collection_double = 10;
     CollectionSInt64 collection_sint64 = 11;
+    SharedMemoryData shared_memory_data = 12;
   }
 }
 
+// Used to provide metadata about shared memory region to read/write data
+message SharedMemoryData {
+  string memory_mapped_file_name = 1;
```

Comment:
nit: rename to `file_name` ?

Reply:
In Windows systems this may not be a file per se but a memory map name.

Reply:
Rename message to `RpcMemoryMap` and  rename `memory_mapped_file_name` to just `name` ?

Reply:
For renaming `memory_mapped_file_name` I agree with you both. However, @pragnagopa I thought the idea was that this approach is not restricted to RPC so renaming the message to `RpcMemoryMap` would go against that. Thoughts?

(referring to the earlier comment you had to move the `SharedMemoryDataTransfer` directory out of the `Rpc` directory and one level up to the `Worker` directory.)

Reply:
This proto file is specific to communicating over gRpc . Type `RpcMemoryMap` would be used specifically for communication between host and worker over gRpc.